src/Text/Parsec/Token.hs

```diff
@@ -390,6 +391,8 @@ makeTokenParser languageDef
     -----------------------------------------------------------
     -- Bracketing
     -----------------------------------------------------------
+    parens, braces, angles, brackets
```

Comment:
why these signatures are required, but not any others in the `where` block?

Reply:
Because MicroHs has `MonoLocalBinds` always on, and these need polymorphic types.

Reply:
Add `{-# LANGUAGE MonoLocalBinds #-}` to module header, and an explaining comment that we need it to allow GHC to catch problematic for MHS code.